### PR TITLE
ux: gateway logs

### PR DESCRIPTION
**Pull request by asoorm**
Fixing some of the startup log order
Organizing some of the startup service checks by moving into the checkup package

### CODE REVIEW

File: gateway/server.go [hunk 1]
```diff
@@ -993,11 +992,14 @@ func Start() {
 		mainLog.Warn("The control_api_port should be changed for production")
 	}
 
-	start()
+	checkup.FileDescriptors()
```
Comment (letzya) [inline gateway/server.go]: I would put everything in one “checkup.Run” public function. This way every time we add something wouldn’t require an update in here and all the checkups would be incapsulated inside “Run”.

Comment (buger) [reply gateway/server.go]: In general, agree. It will require modifying code only in one place if we want to add similar warnings in future.

Comment (buger) [reply gateway/server.go]: In general, agree. It will require modifying code only in one place if we want to add similar warnings in future.

**Comment by asoorm** [comment]
@buger PTAL

**Pull request merged**